src/libraries/Common/tests/System/Security/Cryptography/PlatformSupport.cs

```diff
@@ -11,25 +11,35 @@ namespace Test.Cryptography
 {
     internal static class PlatformSupport
     {
-        private static readonly Dictionary<CngAlgorithm, bool> s_platformCryptoSupportedAlgorithms = new();
+        private const string PlatformCryptoProvider = "Microsoft Platform Crypto Provider";
+        private const string SoftwareKeyStorageProvider = "Microsoft Software Key Storage Provider";
```

Comment:
The Software Key Storage Provider is the default provider, so it should always be functional. I do not think we need to test if the Software Key Storage Provider works.

What you tests do need to check for is if the platform can create a VBS key, which may not work in CI runs, or in down level platforms.

I would probably have a different property under `PlatformSupport` to check and memorize if VBS is available.